stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -139,6 +142,13 @@
     { sender: principal, contract-caller: principal }
     { until-burn-ht: (optional uint) })
 
+;; The signing key slots for a reward cycle
+;; Written by a Stacks node as part of processing the first tenure-start-block after last tenure-start-block in previous reward cycle
+(define-map reward-cycle-signing-keys
```

Comment:
Could you please add a getter?

Reply:
@setzeus what `nextest` command you're running? Can you paste it here so I can look into it?